app/books/templates/books/author_form.html

```diff
@@ -15,7 +15,7 @@ <h1>{{ message }}</h1>
 </form>
 
 {% if form.instance.id %}
```

Comment:
`{% if form.instance.id %}` -> `{% if form.instance.id and request.user.is_authenticated%}`